torchvision/io/image.py

```diff
@@ -50,6 +50,13 @@
 
 
 class ImageReadMode(Enum):
+    """
+    Use `ImageReadMode.UNCHANGED` for loading
+    the image as-is, `ImageReadMode.GRAY` for converting to grayscale,
+    `ImageReadMode.GRAY_ALPHA` for grayscale with transparency,
+    `ImageReadMode.RGB` for RGB and `ImageReadMode.RGB_ALPHA` for
+    RGB with transparency. Default: `ImageReadMode.UNCHANGED`
```

Comment:
Before explaining what each value does, perhaps we should mention where this enum is used.

Also note that the enum does not have a "Default" value. The Default values are applicable on the methods that use them. 

Finally I think it would be best to remove the duplicate documentation from the methods (where we explain the values one by one) and point the reader to this class for more info. 

Let me know what you think.

Reply:
Here is the updated documentation for enum:
```
Support for various modes while reading images.

Use `ImageReadMode.UNCHANGED` for loading the image as-is, 
`ImageReadMode.GRAY` for converting to grayscale,
`ImageReadMode.GRAY_ALPHA` for grayscale with transparency,
`ImageReadMode.RGB` for RGB and `ImageReadMode.RGB_ALPHA` for
RGB with transparency.
```
Is it better now?

Yes, I agree with you. I will remove the duplicated from the method's documentation. Also, I will mention the default value in the methods itself (I have removed that from enum's documentation) and point the reader to the class for more information.

Do let me know if there is any concern. Thanks!

Reply:
Sounds good, please apply your proposed changes. :)

Reply:
I have applied the proposed changes. Please have a look at them and let me know if there is any concern. Thank you!